### PR TITLE
detect metamask network & changed send amount view for metamask

### PR DESCRIPTION
**Pull request by rumi-art**

### CODE REVIEW

**Review by orbitmechanic — APPROVED** [review]
Nice!  Straight asking Moralis for the networkId.  Makes sense.

**Comment by rumi-art** [comment]
Please check this with #80 issue together @orbitmechanic

File: website/src/components/Bits/FromSelect.jsx [hunk 1]
```diff
@@ -21,53 +23,61 @@ export const FromSelect = () => {
     };
   }, [setFromToken]);
 
-  const handleChange = (e) => {
+  const handleChange = async (e) => {
+    await Moralis.enable();
+    const id = await Moralis.getChainId();
```
Comment (orbitmechanic) [inline website/src/components/Bits/FromSelect.jsx:28]: The key to the operation.

File: website/src/components/Bits/FromSelect.jsx [hunk 2]
```diff
@@ -21,53 +23,61 @@ export const FromSelect = () => {
...
+    if (id !== 137) {
+      setFromToken();
+      setToToken();
+      setDialog('Switch network to Polygon');
```
Comment (orbitmechanic) [inline website/src/components/Bits/FromSelect.jsx:32]: This should call a function here to actually do the switch autonomously.

File: website/src/components/Blocks/SendPanel.jsx [hunk 3]
```diff
@@ -27,7 +27,7 @@ export const SendPanel = () => {
         <FromSelect />
         {!!fromSymbol && (
           <>
-            <AmountSelect />
+            <AmountSelect type='send'/>
```
Comment (orbitmechanic) [inline website/src/components/Blocks/SendPanel.jsx:30]: Interesting new parameter...

**Review by orbitmechanic — APPROVED** [review]
Interesting additions. I had not noticed the type parameter in the AmountSelect before.  I'll have to read through that.  Thank you!

**Pull request merged**